src/domain/submission/services/submission-service.ts

```diff
@@ -79,7 +81,38 @@ export class SubmissionService {
 
         submission.status = SubmissionStatus.MECA_EXPORT_PENDING;
         await this.submissionRepository.update(submission);
-
+        const toEmail = submission.author?.email;
+        const data = {
+            html: `<p>Dear ${submission.author?.firstName}</p>,
```

Comment:
Wondering if this should be an imported value from somewhere easier to find and change (config or something?)